src/applications/app.helpers.js

```diff
@@ -22,7 +23,7 @@ export function isntActive(app) {
 }
 
 export function isLoaded(app) {
-  return app.status !== NOT_LOADED && app.status !== LOADING_SOURCE_CODE;
+  return app.status !== NOT_LOADED && app.status !== LOADING_SOURCE_CODE && app.status !== LOAD_ERROR;
```

Comment:
While we're at it, add in `&& app.status !== SKIP_BECAUSE_BROKEN`??

I think that not having SKIP_BECAUSE_BROKEN in this function has worked out because we filter out the skipped applications before calling the isLoaded function. But it should be included for completeness, I think, of this function.

Reply:
Technically though, might an app be loaded but still broken?

Reply:
ah good point. I guess SKIP_BECAUSE_BROKEN doesn't indicate things one way or the other. No need to change this then 😄